tests/test_form.py

```diff
@@ -0,0 +1,14 @@
+import os
+import unittest
+from py4web import response, Field
+from py4web.utils.form import Form
+
+
+class FormTest(unittest.TestCase):
+    def test_form(self):
+        table = [Field("name")]
+        form_name = 'testing_form'
+        f = Form(table,form_name=form_name)
```

Comment:
This test fails for me because Form still tries parse a request object that does not exist. I think this may fix it:

```
def setUp(self):
    request.environ['wsgi.input'] = BytesIO()
    request.environ['wsgi.input'].write("")
    request.environ['wsgi.input'].seek(0)
```